src/rc_event.c

```diff
@@ -38,7 +38,9 @@ static int rc_event_kthread(void *arg);
 extern int rc_srb_seq_num;
 
 rc_event_thread_t rc_event_thread;
-
+#if 0
```

Comment:
Are these `#if 0` sections by you (MarkLTZ) or AMD?

Reply:
by the AMD driver. I just fixed some ugly trailing spaces.